src/Handlebars/Context.php

```diff
@@ -125,26 +149,6 @@ public function last()
         return end($this->stack);
     }
 
-    /**
-     * Get the index of current section item.
-     *
-     * @return mixed Last index
-     */
-    public function lastIndex()
```

Comment:
Why are you dropping these?

Reply:
Oh, forgot to drop a comment here. These are being removed because the only usage has been replaced with the [variable retrieval within Template.php](https://github.com/salesforce/handlebars-php/pull/3/files#diff-6f379a40304cb067449032067feac1fbR348). The data variables support retrieving the last index (`{{@index}}`) or relative pathing (`{{@../index}}`) so we can use the same function for both situations.

Reply:
The same applies for the `lastKey` function below.

Reply:
Gotcha, I wonder if we run into any issues by virtue of changing the public API with this though?

Reply:
Yeah, I didn't think about that so it's a good call out. I've updated the code to put the new data variables behind a feature flag. It doesn't quite match how Handlebars JS works but it solves the problem of opting-in to the new behavior.

To enable data variables, the user must pass in `enableDataVariables` when constructing the Handlebars instance. By default this is set to `false`.

```php
$engine = new \Handlebars\Handlebars(array(
  'enableDataVariables'=> true,
));
```